src/common/io/http/request.c

```diff
@@ -243,7 +243,7 @@ httpRequestError(const HttpRequest *this, HttpResponse *response)
     ASSERT(response != NULL);
 
     // Error code
-    String *error = strCatFmt(strNew(), "HTTP request failed with %u", httpResponseCode(response));
+    String *error = strCatFmt(strNew(), "HTTP %s request failed with %u", strZ(httpRequestVerb(this)), httpResponseCode(response));
```

Comment:
I think think would be better at line 255, i.e.:
```
strCatFmt(error, "\n%s %s", strZ(httpRequestVerb(this)), strZ(httpRequestPath(this)));
```

Reply:
I moved the output to that location.

Reply:
That looks good to me.